src/Facebook/InstantArticles/Client/ServerMessage.php

```diff
@@ -73,8 +73,6 @@ public static function fromLevel($level, $message)
         if ($validLevel) {
             return new self($level, $message);
         } else {
-            \Logger::getLogger('facebook-instantarticles-client')
-                ->info('Unknown message level "$level". Are you using the last SDK version?');
```

Comment:
Will this file functions will be used over Transformer.php implementations?

Reply:
No.
All the package `/Client` is on SDK for making it easier to call all APIs for the IA ingestion. [See more here](https://developers.facebook.com/docs/instant-articles/api).